README.md

```diff
@@ -4,23 +4,24 @@
      title="Philosopher’s stone, logo of PostCSS"
      src="http://postcss.github.io/postcss/logo-leftp.png">
 
-Boilerplate repoitory to start PostCSS plugin by few steps.
+This is the boilerplate repository used to create new PostCSS plugins in a few 
+steps.
 
-1. Clone this reposity:
+1. Clone this repository:
 
    ```sh
   git clone https://github.com/postcss/postcss-plugin-boilerplate.git
    ```
 
-2. Execute wizard script. it will ask you a few question and will fill all
+2. Execute the wizard script. It will ask you a few question and will fill all
    files with your data.
 
     ```sh
    ./postcss-plugin-boilerplate/start
     ```
 
-3. Now it is repository of your plugin with clean Git history.
-   Create GitHub repositpory and push your project there.
+3. Now it is the repository of your plugin with a clean Git history.
```

Comment:
oh, and github help spell it this way too https://help.github.com/articles/https-cloning-errors/